cmake/exiv2.pc.in

```diff
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=@libdir_for_pc_file@
-includedir=@includedir_for_pc_file@
+libdir=@LIBDIR@
+includedir=@INCLUDEDIR@
```

Comment:
These shortened variables are too generic and there is risk they are used by some other project that could e.g. include the Exiv2 one...

Reply:
I disagree but renamed. It's unfortunate CMake doesn't have a pkgconfig writer like meson's.